src/main/java/org/jetbrains/bsp/bazel/server/bsp/BspIntegration.java

```diff
@@ -0,0 +1,60 @@
+package org.jetbrains.bsp.bazel.server.bsp;
+
+import ch.epfl.scala.bsp4j.BuildClient;
+import io.grpc.Server;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.util.concurrent.ExecutorService;
+import org.eclipse.lsp4j.jsonrpc.Launcher;
+
+public class BspIntegration {
```

Comment:
Maybe simply _BspIntegrationData_? Or do you have some idea?

Reply:
`BspIntegrationData` works for me

Reply:
or `BspInitializationData` because as I remember it is used in the initalization

Reply:
The idea was that this is an object for exchanging data needed to integrate various components with each other during the initialisation because of complicated dependencies between them, so I'll go with _Integration_.

Reply:
okey then